docs/python3-mac-installation.md

```diff
@@ -0,0 +1,13 @@
+# Latest Python3 installation on macOS
+
+_Note:_ This guide does not cover `homebrew` based Python3 installation.
```

Comment:
Highlight the reason for those steps: the python version bundled with mac out the box is not suitable